npm install doesn't seem to work on Windows

Using latest master
node v7.10.0
npm  v4.2.0
Windows 8.1

```
...
> node scripts/generate.js && lerna bootstrap --hoist

core: yo theia:extension --force
cpp: yo theia:extension --force
editor: yo theia:extension --force
filesystem: yo theia:extension --force
java: yo theia:extension --force
languages: yo theia:extension --force
monaco: yo theia:extension --force
navigator: yo theia:extension --force
preferences: yo theia:extension --force
python: yo theia:extension --force
terminal: yo theia:extension --force
workspace: yo theia:extension --force
browser: yo theia:browser --force
electron: yo theia:electron --force
core: spawn yo ENOENT
cpp: spawn yo ENOENT
editor: spawn yo ENOENT
filesystem: spawn yo ENOENT
java: spawn yo ENOENT
languages: spawn yo ENOENT
monaco: spawn yo ENOENT
navigator: spawn yo ENOENT
preferences: spawn yo ENOENT
python: spawn yo ENOENT
terminal: spawn yo ENOENT
workspace: spawn yo ENOENT
browser: spawn yo ENOENT
electron: spawn yo ENOENT
lerna info version 2.0.0
lerna info versioning independent
lerna info Bootstrapping 0 packages
lerna info lifecycle preinstall
lerna info Installing external dependencies
lerna info hoist Finished installing in root
lerna info Symlinking packages and binaries
lerna info lifecycle postinstall
lerna info lifecycle prepublish
lerna success Bootstrapped 0 packages

> @theia/parent@0.0.0 clean C:\Users\Marc-Andre\git\theia
> lerna run clean --parallel

lerna info version 2.0.0
lerna info versioning independent
lerna WARN No packages found with the npm script 'clean'
lerna info run in 0 package(s): npm run clean

> @theia/parent@0.0.0 build C:\Users\Marc-Andre\git\theia
> lerna run build --stream

lerna info version 2.0.0
lerna info versioning independent
lerna WARN No packages found with the npm script 'build'
```

Then when I try to go to examples/electron:
```
$ npm run start
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_module                                       s\\npm\\bin\\npm-cli.js" "run" "start"
npm ERR! node v7.10.0
npm ERR! npm  v4.2.0

npm ERR! missing script: start
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\Marc-Andre\AppData\Roaming\npm-cache\_logs\2017-07-18T15_02_36_217Z-d                                       ebug.log
```

--- a/generator-theia/src/common/app-package-generator.ts
+++ b/generator-theia/src/common/app-package-generator.ts
@@ -46,7 +46,7 @@ export class AppPackageGenerator extends AbstractGenerator {
                 "cold:start": "yarn run clean && yarn run start",
                 "build": "yarn run build:frontend && yarn run build:backend",
                 "build:frontend": "webpack",
-                "build:backend": `cp ${this.srcGen()}/frontend/index.html lib`,
+                "build:backend": `shx cp ${this.srcGen()}/frontend/index.html lib`,
                 "watch": "yarn run build:frontend && webpack --watch",
                 ...scripts,
                 ...this.model.pck.scripts
@@ -61,6 +61,7 @@ export class AppPackageGenerator extends AbstractGenerator {
                 "css-loader": "^0.28.1",
                 "file-loader": "^0.11.1",
                 "source-map-loader": "^0.2.1",
+                "shx": "^0.2.2",
                 "url-loader": "^0.5.8",
                 "font-awesome-webpack": "0.0.5-beta.2",
                 "less": "^2.7.2",

